packages/phpstan-rules/packages/nette/tests/Rules/LatteCompleteCheckRuleTemplateSetup/LatteCompleteCheckRuleTemplateSetupTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symplify\PHPStanRules\Nette\Tests\Rules\LatteCompleteCheckRuleTemplateSetup;
+
+use PHPStan\Rules\Rule;
+use Symplify\PHPStanExtensions\Testing\AbstractServiceAwareRuleTestCase;
+use Symplify\PHPStanRules\Nette\Rules\LatteCompleteCheckRule;
+
+/**
+ * @extends AbstractServiceAwareRuleTestCase<LatteCompleteCheckRule>
+ */
+final class LatteCompleteCheckRuleTemplateSetupTest extends AbstractServiceAwareRuleTestCase
+{
+    public function testRule(): void
+    {
+        $fileToClass = [
+            __DIR__ . '/Fixtures/PropertyReadTemplate/ExampleControl.php' => \Symplify\PHPStanRules\Nette\Tests\Rules\LatteCompleteCheckRuleTemplateSetup\Fixtures\PropertyReadTemplate\ExampleControl::class,
+            __DIR__ . '/Fixtures/NoAdditionalPropertyRead/ExampleControl.php' => \Symplify\PHPStanRules\Nette\Tests\Rules\LatteCompleteCheckRuleTemplateSetup\Fixtures\NoAdditionalPropertyRead\ExampleControl::class,
+            __DIR__ . '/Fixtures/GetTemplateAndReplaceExtension/ExampleControl.php' => \Symplify\PHPStanRules\Nette\Tests\Rules\LatteCompleteCheckRuleTemplateSetup\Fixtures\GetTemplateAndReplaceExtension\ExampleControl::class,
+            __DIR__ . '/Fixtures/RenderWithParameters/ExampleControl.php' => \Symplify\PHPStanRules\Nette\Tests\Rules\LatteCompleteCheckRuleTemplateSetup\Fixtures\RenderWithParameters\ExampleControl::class,
+            __DIR__ . '/Fixtures/TemplateAsVariableAndRenderToStringWithParameters/ExampleControl.php' => \Symplify\PHPStanRules\Nette\Tests\Rules\LatteCompleteCheckRuleTemplateSetup\Fixtures\TemplateAsVariableAndRenderToStringWithParameters\ExampleControl::class,
+        ];
+
+        foreach ($fileToClass as $file => $class) {
+            $this->analyse([$file], [
+//                [
```

Comment:
@TomasVotruba any idea why these errors are not identified?

Reply:
Is the file rendered correctly?
It the responsible PHPStan rule registered and active in test config?


Btw, I can't see which errors are related to which file. Seems like each file reports all the errors :D 

Could you pair it error + file like all the other test we have here?

Reply:
> Is the file rendered correctly?

Yes, it is

> It the responsible PHPStan rule registered and active in test config?

Ah, I thought there are all active :) I have to check it again

> Btw, I can't see which errors are related to which file. Seems like each file reports all the errors :D
> Could you pair it error + file like all the other test we have here?

Exactly :) one template for all controls. It is about Template setup with different ways not about control itself - probably these Fixtures will be removed and I will add some service for extracting templates / parameters with tests

Reply:
> It is about Template setup with different ways not about control itself - probably these Fixtures will be removed and I will add some service for extracting templates / parameters with tests

I see :+1: The foreach confused me. Could you separate it to explicit calls?